ktor-http/common/test/io/ktor/tests/http/DataUrlTest.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.http
+
+import io.ktor.http.*
+import kotlin.test.*
+
+class DataUrlTest
```

Comment:
Should we move the new tests from `UrlTest.kt` to this new class `DataUrlTest.kt`?